src/sentry/integrations/utils/commit_context.py

```diff
@@ -107,4 +108,4 @@ def find_commit_context_for_event(
         if commit_context:
             result.append((commit_context, code_mapping))
 
-    return result
+    return result, install
```

Comment:
`install` will only ever provide the last valid installation in the `code_mappings`, do we want this? If so, could we refactor it out of the loop?

Reply:
i'm not sure it matter which installation we get? we're only using it to call the github API
cc @AniketDas-Tekky

Reply:
Yeah, we only need it for the client

Reply:
i will find one that's not none rather than only get the last one, though